index.js

```diff
@@ -108,7 +108,7 @@ exports.build = function build(argv, callback) {
         install = true;
         break;
       case 'b':
-        bundle = true;
+        console.error('Warning, the --bundle option has been removed');
```

Comment:
`the bundle option now does nothing, it should not be used`? Its not removed if you just used it.